seatunnel-server/seatunnel-server-common/src/main/java/org/apache/seatunnel/server/common/SeatunnelErrorEnum.java

```diff
@@ -32,8 +32,8 @@ public enum SeatunnelErrorEnum {
 
     USERNAME_PASSWORD_NO_MATCHED(
             10007,
-            "username and password no matched",
-            "The user name and password do not match, please check your input"),
+            "username and password no matched or user is disabled.",
```

Comment:
please fix the typo, 

```suggestion
            "username and password not matched or user is disabled."

Reply:
The existing message had the typo. I retained the same message initially, but it can be corrected. I have now corrected it.

Reply:
The existing message had the typo. I retained the same message initially, but it can be corrected. I have now corrected it.